btax/parameters.py

```diff
@@ -68,6 +68,18 @@ def translate_param_names():
 	# btax_depr_allyr_gds_Switch
 	# btax_depr_allyr_tax_Switch
 
+	class_list = [3, 5, 7, 10, 15, 20, 25, 27.5, 39]
+
+	user_bonus_deprec={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
+					'7':btax_depr_7yr_exp, '10':btax_depr_10yr_exp,
+					'15':btax_depr_15yr_exp, '20':btax_depr_20yr_exp,
+					'25':btax_depr_25yr_exp, '27.5':btax_depr_27_5yr_exp,
+					'39':btax_depr_39yr_exp}
+	user_deprec_system={'3':btax_depr_3yr_exp, '5':btax_depr_5yr_exp,
```

Comment:
I agree.  I can build off of this some.  I'm separating some of `get_params` into a 2 functions, one to get the base params and one to get the derived params from there.  That will make it easier to use in different contexts (UI vs command line).